msr/msre/pipe_cardinal/tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
+  [syntax_check_cardinal]
+    type = RunApp
+    input = cardinal.i
+    check_input = True
+    executable_pattern = 'cardinal*|combined*'
+    cli_args = "MultiApps/active='' Transfers/active=''"
+  []
+  [syntax_check_nek]
+    type = RunApp
+    input = nek.i
+    check_input = True
+    executable_pattern = 'cardinal*'
+    min_parallel = 16
+    skip = 'Not enough memory #438'
+  []
+[]
```

Comment:
the nekRS model has 494K elements, with the order `p=5`, the resulting number of dofs is ~62 million. so running a fully coupled case involving the CFD simulation may not be very feasible with 16 CPU processes. Having said that, a test of Solid Mechanics only calculation might be possible. Let me see what we can do.

Reply:
Hi Guillaume, I have a question related to the regression test. Currently, I'm storing the moose mesh file using git-lfs. Do I need to convert it to a regular file for Civet tests to access it? It seems like the Cardinal syntax check test isn't able to use it as is.
```
*** ERROR ***
data/civet1/build/virtual_test_bed/msr/msre/pipe_cardinal.syntax_check_cardinal: /data/civet1/build/virtual_test_bed/msr/msre/pipe_cardinal/./solid_hex8.exo appears to be a Git-LFS pointer. Make sure you have "git-lfs" installed so that you may pull this file.
```

Reply:
> what is the expectation of the upcoming testing server? particularly, would GPU be supported?

yes we should have some GPUs

> . Do I need to convert it to a regular file for Civet tests to access it?

no, we just need to turn on the LFS file downloading in the civet_recipes
I can do that tonight

Reply:
Sounds good, thanks!

Reply:
ok the change is merged. in 10 minutes or so it should download the exodus file

Reply:
Looks like this new Cardinal model passed the testing but several other cases failed. 
```
[4.184s]       OK data/civet1/build/virtual_test_bed/msr/msre/pipe_cardinal.syntax_check_cardinal SYNTAX PASS
...
[251.5s]       OK data/civet1/build/virtual_test_bed/msr/msre/pipe_cardinal.solid_mechanics [FINISHED]
```
The failed tests also complained about the mesh files stored in git-lfs.

Reply:
then this model should be ready to go. thanks for all the help and suggestions.

Reply:
I ll review tomorrow again. looking good!